install/Makefile

```diff
@@ -74,7 +74,7 @@ APACHE_PROGRAM           = /usr/sbin/apache2
 
 # For testing proxying of an external domain, this represents the domain we
 # are proxying from.
-PAGESPEED_TEST_HOST ?= modpagespeed.com
+PAGESPEED_TEST_HOST ?= selfsigned.modpagespeed.com
```

Comment:
@morlovich re:"So this is a bit surprising, since selfsigned. was only meant for the tests that explicitly wanted a self-signed cert rather than proper one?" 

I can see the surprise, but I thought selfsigned.modpagespeed.com was used in other places too:
https://github.com/pagespeed/mod_pagespeed/search?utf8=%E2%9C%93&q=selfsigned&type=
(selfsigned.modpagespeed.com also serves plain http). 
Should we perhaps add a separate, dedicated hostname for this that is less confusing?